tests/h/search/old_query_test.py

```diff
@@ -83,33 +89,6 @@ def test_limit_missing(self):
 
         assert q["size"] == LIMIT_DEFAULT
 
-    def test_sort_is_by_updated(self):
-        """Sort defaults to "updated"."""
-        builder = query.Builder(ES_VERSION)
-
-        q = builder.build({})
-
-        sort = q["sort"]
-        assert len(sort) == 1
-        assert list(sort[0].keys()) == ["updated"]
-
-    def test_with_custom_sort(self):
-        """Custom sorts are returned in the query dict."""
-        builder = query.Builder(ES_VERSION)
-
-        q = builder.build({"sort": "title"})
-
-        assert q["sort"] == [{'title': {'unmapped_type': 'boolean', 'order': 'desc'}}]
-
-    def test_order_defaults_to_desc(self):
-        """'order': "desc" is returned in the q dict by default."""
-        builder = query.Builder(ES_VERSION)
-
-        q = builder.build({})
-
-        sort = q["sort"]
-        assert sort[0]["updated"]["order"] == "desc"
-
     def test_with_custom_order(self):
```

Comment:
This test can be removed also as it is covered by test_it_sorts_annotations in the new tests already.